### PR TITLE
Adding 10.7.0 changelog

### PR DESCRIPTION
**Pull request by mike-plummer**

### CODE REVIEW

File: content/_changelogs/10.7.0.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- New informational banners will display instructions to assist with connecting projects to the Cypress Dashboard and recording a first test run. Users who don't plan to use the Cypress Dashboard can dismiss the banners to not see them in the future. Addresses [#22933](https://github.com/cypress-io/cypress/issues/22933), [#22934](https://github.com/cypress-io/cypress/issues/22934), [#22798](https://github.com/cypress-io/cypress/issues/22798), and [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the "Create Spec from Component" capability introduced in v10.5.0. Addresses [#23298](https://github.com/cypress-io/cypress/issues/23298)
+- Angular component testing projects can provide a custom configuration to the Webpack dev server using the `projectConfig` key. This enables support for projects that don't adhere to Angular CLI conventions. Addresses [#23161](https://github.com/cypress-io/cypress/issues/23161)
```
Comment (rachelruderman) [inline content/_changelogs/10.7.0.md:12]: Heads up, it looks like this issue isn't inside a release ([the PR that fixes it](https://github.com/cypress-io/cypress/pull/23494) isn't either)

File: content/_changelogs/10.7.0.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Adds [Svelte](https://svelte.dev/) component testing support. Addresses [#23431](https://github.com/cypress-io/cypress/issues/23431) and [#23432](https://github.com/cypress-io/cypress/issues/23432).
+- New informational banners will display instructions to assist with connecting projects to the Cypress Dashboard and recording a first test run. Users who don't plan to use the Cypress Dashboard can dismiss the banners to not see them in the future. Addresses [#22933](https://github.com/cypress-io/cypress/issues/22933), [#22934](https://github.com/cypress-io/cypress/issues/22934), [#22798](https://github.com/cypress-io/cypress/issues/22798), and [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the "Create Spec from Component" capability introduced in v10.5.0. Addresses [#23298](https://github.com/cypress-io/cypress/issues/23298)
```
Comment (rachelruderman) [inline content/_changelogs/10.7.0.md:11]: This one and its PR also aren't inside a release -- @tbiethman we should add the release to the issue when it's missing, right?

File: content/_changelogs/10.7.0.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- Angular component testing projects can provide a custom configuration to the Webpack dev server using the `projectConfig` key. This enables support for projects that don't adhere to Angular CLI conventions. Addresses [#23161](https://github.com/cypress-io/cypress/issues/23161)
+- The Cypress `clock` can now be set to a new time using the `setSystemTime` function. Addresses [#15424](https://github.com/cypress-io/cypress/issues/15424)
```
Comment (rachelruderman) [inline content/_changelogs/10.7.0.md:13]: Same release dealio

File: content/_changelogs/10.7.0.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+**Bugfixes:**
+
+- Fixed an issue where Cypress could crash if a websocket upgrade request was made over HTTPS. Fixes [#22217](https://github.com/cypress-io/cypress/issues/22217)
+- Fixed an issue where filtering with the `--spec` run mode option would not find any specs if the project was located at the root directory of a file system. Fixes [#23380](https://github.com/cypress-io/cypress/issues/23380)
+- Fixed an issue where projects using Angular v14.2.0 and above would be unable to run component tests. Fixes [#23585](https://github.com/cypress-io/cypress/issues/23585)
+- Fixed an issue where snapshots performed in XHR requests could reflect the primary domain instead of current domain. Fixes [#21496](https://github.com/cypress-io/cypress/issues/21496)
+- Introduced a new `GITHUB_RUN_ATTEMPT` environment variable when claiming specs from Cypress Cloud to address a potential scenario that would cause invalid Github Action requests to be accepted. Addressed in [#23445](https://github.com/cypress-io/cypress/pull/23445)
```
Comment (rachelruderman) [inline content/_changelogs/10.7.0.md:27]: What does "claiming specs from Cypress Cloud" mean? Is claiming maybe a typo? Or just lingo I'm unfamiliar with 😝

File: content/_changelogs/10.7.0.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- Fixed an issue where an internal Typescript type was exposed globally. Fixes [#23333](https://github.com/cypress-io/cypress/pull/23333)
```
Comment (rachelruderman) [inline content/_changelogs/10.7.0.md:28]:
```suggestion
- Fixed an issue where an internal TypeScript type was exposed globally. Fixes [#23333](https://github.com/cypress-io/cypress/pull/23333)
```

File: content/_changelogs/10.7.0.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+## 10.7.0
+
+_Released 8/30/2022_
+
+
+**Features:**
+
+- Re-introducing the experimental [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio). Cypress Studio provides a visual way to generate tests within Cypress by recording interactions against the application under test. Cypress Studio must be enabled via the `experimentalStudio` flag and is currently not supported in component tests. Addresses [#23461](https://github.com/cypress-io/cypress/issues/23461)
```
Comment (lmiller1990) [inline content/_changelogs/10.7.0.md:8]: I think changelogs tend to be written past tense... eg "re-introduced" etc (just based on our other change logs, not sure if there's a hard/fast rule)

File: content/_changelogs/10.7.0.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+
+**Miscellaneous:**
+
+- Display a default icon in the browser selection menu for browsers without bundled icons. Fixes [#21766](https://github.com/cypress-io/cypress/issues/21766)
+
+
+**Dependency Updates:**
+
+- Upgraded the bundled Typescript version shipped with Cypress from `4.2.3` to `4.7.4`. Addressed in [#22252](https://github.com/cypress-io/cypress/pull/22252).
```
Comment (rachelruderman) [inline content/_changelogs/10.7.0.md:39]:
```suggestion
- Upgraded the bundled TypeScript version shipped with Cypress from `4.2.3` to `4.7.4`. Addressed in [#22252](https://github.com/cypress-io/cypress/pull/22252).
```

Also, I think this is the wrong link -- this link is to a PR released in 10.2.0 with an Electron version bump and arm64/aarch64 systems

File: content/_changelogs/10.7.0.md (hunk 6 above)
Comment (BlueWinds) [reply content/_changelogs/10.7.0.md:8]: At least a more passive present tense.

```suggestion
- Re-introduces the experimental [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio). Cypress Studio provides a visual way to generate tests within Cypress by recording interactions against the application under test. Cypress Studio is available only in end-to-end tests, and must be enabled via the `experimentalStudio` flag. Addresses [#23461](https://github.com/cypress-io/cypress/issues/23461)
```

File: content/_changelogs/10.7.0.md (hunk 7 above)
Comment (rachelruderman) [reply content/_changelogs/10.7.0.md:39]: I think this is the right link: https://github.com/cypress-io/cypress/pull/23523 (and needs to be tagged in a release)

File: content/_changelogs/10.7.0.md (hunk 4 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:27]: 🤷‍♂️ I was having trouble converting the User Facing Changelog entry from that PR into something more digestable. I'll ping Jaime in the morning to see if he can dumb it down for me

File: content/_changelogs/10.7.0.md (hunk 7 above)
Comment (rachelruderman) [reply content/_changelogs/10.7.0.md:39]: Final thought -- this was an update to a dev dependency, as opposed to a dependency or a peer dependency. Should it be in the changelog?

File: content/_changelogs/10.7.0.md (hunk 1 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:12]: Dang, looks like I missed marking the release on a couple issues. Tagged

File: content/_changelogs/10.7.0.md (hunk 2 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:11]: Tagged issue, updating link to point to issue instead of PR

File: content/_changelogs/10.7.0.md (hunk 3 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:13]: Donezo

**Review by lmiller1990 — APPROVED** [review]
All my comments were addressed 👍

File: content/_changelogs/10.7.0.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
+## 10.7.0
+
+_Released 8/30/2022_
+
+**Features:**
+
+- Re-introduces the experimental
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:7]: @mike-plummer I remember mention of a blog post for this? If we have one, we should call it out. Here is an 10.3.0 is an example where we do this: https://docs.cypress.io/guides/references/changelog#10-3-0

File: content/_changelogs/10.7.0.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio).
+  Cypress Studio provides a visual way to generate tests within Cypress by
+  recording interactions against the application under test. Cypress Studio is
+  available only in end-to-end tests, and must be enabled via the
+  `experimentalStudio` flag. Addresses
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:12]:
```suggestion
  `e2e.experimentalStudio` flag. Addresses
```

File: content/_changelogs/10.7.0.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+  [#23461](https://github.com/cypress-io/cypress/issues/23461)
+- Adds [Svelte](https://svelte.dev/) component testing support. Addresses
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:14]: Is this experimental / beta support? Also I assume we should have closed out & referenced the epic for this work? https://github.com/cypress-io/cypress/issues/23037

if the epic isn't correct, I'd update this to reference https://github.com/cypress-io/cypress/pull/23553 that added this code to develop since these issues were tasks to completing the single feature and we don't need to be as granular for the initial release.

File: content/_changelogs/10.7.0.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+  [#23431](https://github.com/cypress-io/cypress/issues/23431) and
+  [#23432](https://github.com/cypress-io/cypress/issues/23432).
+- The "Latest Runs" and "Average Duration" columns in the specs list now provide
+  actions when hovered over to assist with the process of connecting the project
+  to the Cypress Dashboard. Addresses
+  [#22932](https://github.com/cypress-io/cypress/issues/22932)
+- New informational banners will display instructions to assist with connecting
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:21]:
```suggestion
- Informational banners were added to display instructions to assist with connecting
```

File: content/_changelogs/10.7.0.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+  projects to the Cypress Dashboard and recording a first test run. Users who
+  don't plan to use the Cypress Dashboard can dismiss the banners to not see
+  them in the future. Addresses
+  [#22933](https://github.com/cypress-io/cypress/issues/22933),
+  [#22934](https://github.com/cypress-io/cypress/issues/22934),
+  [#22798](https://github.com/cypress-io/cypress/issues/22798), and
+  [#22935](https://github.com/cypress-io/cypress/issues/22935)
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:28]:
```suggestion
  [#22935](https://github.com/cypress-io/cypress/issues/22935).
```

File: content/_changelogs/10.7.0.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+  available only in end-to-end tests, and must be enabled via the
+  `experimentalStudio` flag. Addresses
+  [#23461](https://github.com/cypress-io/cypress/issues/23461)
+- Adds [Svelte](https://svelte.dev/) component testing support. Addresses
+  [#23431](https://github.com/cypress-io/cypress/issues/23431) and
+  [#23432](https://github.com/cypress-io/cypress/issues/23432).
+- The "Latest Runs" and "Average Duration" columns in the specs list now provide
+  actions when hovered over to assist with the process of connecting the project
+  to the Cypress Dashboard. Addresses
+  [#22932](https://github.com/cypress-io/cypress/issues/22932)
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:20]:
```suggestion
  [#22932](https://github.com/cypress-io/cypress/issues/22932).
```

File: content/_changelogs/10.7.0.md [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+
+**Features:**
+
+- Re-introduces the experimental
+  [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio).
+  Cypress Studio provides a visual way to generate tests within Cypress by
+  recording interactions against the application under test. Cypress Studio is
+  available only in end-to-end tests, and must be enabled via the
+  `experimentalStudio` flag. Addresses
+  [#23461](https://github.com/cypress-io/cypress/issues/23461)
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:13]:
```suggestion
  [#23461](https://github.com/cypress-io/cypress/issues/23461).
```

File: content/_changelogs/10.7.0.md [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+  projects to the Cypress Dashboard and recording a first test run. Users who
+  don't plan to use the Cypress Dashboard can dismiss the banners to not see
+  them in the future. Addresses
+  [#22933](https://github.com/cypress-io/cypress/issues/22933),
+  [#22934](https://github.com/cypress-io/cypress/issues/22934),
+  [#22798](https://github.com/cypress-io/cypress/issues/22798), and
+  [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the
+  "Create Spec from Component" capability introduced in v10.5.0. Addresses
+  [#23071](https://github.com/cypress-io/cypress/issues/23071)
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:31]:
```suggestion
  [#23071](https://github.com/cypress-io/cypress/issues/23071).
```

File: content/_changelogs/10.7.0.md (hunk 8 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:7]: Pinged Jess to see about the potential blog post, waiting...

File: content/_changelogs/10.7.0.md (hunk 10 above)
Comment (emilyrohrbough) [reply content/_changelogs/10.7.0.md:14]: I just found [this guide](https://docs.google.com/document/d/1pOZ0Ntm4b1Eev_iY5vLBPhOxx66Z8Njrvvg6WMCvWRI/edit#heading=h.ok2apo8hzldd) recently that DX put together, but it calls out ct testing should be capitalized as `Component Testing`. I noticed in a few places.

File: content/_changelogs/10.7.0.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+  dom after a validation function runs. This means a `cy.visit()` command is no
+  longer required after running `cy.session()` when a validation function is
+  used. Addresses [#22368](https://github.com/cypress-io/cypress/issues/22368).
+
+**Bugfixes:**
+
+- Fixed an issue where Cypress could crash if a websocket upgrade request was
+  made over HTTPS. Fixes
+  [#22217](https://github.com/cypress-io/cypress/issues/22217)
+- Fixed an issue where filtering with the `--spec` run mode option would not
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:52]:
```suggestion
- Fixed an issue where filtering with the `--spec` CLI flag in run mode option would not
```

File: content/_changelogs/10.7.0.md [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+  find any specs if the project was located at the root directory of a file
+  system. Fixes [#23380](https://github.com/cypress-io/cypress/issues/23380)
+- Fixed an issue where projects using Angular v14.2.0 and above would be unable
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:55]:
```suggestion
- Fixed an issue where projects using Angular `v14.2.0` and above would be unable
```

File: content/_changelogs/10.7.0.md [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+  to run component tests. Fixes
+  [#23585](https://github.com/cypress-io/cypress/issues/23585)
+- Fixed an issue where snapshots performed in XHR requests could reflect the
+  primary domain instead of current domain. Fixes
+  [#21496](https://github.com/cypress-io/cypress/issues/21496)
+- Introduced a new `GITHUB_RUN_ATTEMPT` environment variable when claiming specs
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:61]: Can we link to this the EVN docs?

File: content/_changelogs/10.7.0.md [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+
+- Display a default icon in the browser selection menu for browsers without
+  bundled icons. Fixes
+  [#21766](https://github.com/cypress-io/cypress/issues/21766)
+
+**Dependency Updates:**
+
+- Upgraded the bundled TypeScript version shipped with Cypress from `4.2.3` to
+  `4.7.4`. Addressed in
+  [#22252](https://github.com/cypress-io/cypress/pull/22252).
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:81]:
```suggestion
  [#23523](https://github.com/cypress-io/cypress/pull/23523).
```

Comment (emilyrohrbough) [reply content/_changelogs/10.7.0.md:81]: @ZachJW34 I feel like I'm misremembering, was this bump to enable typescript & node 18 support?

File: content/_changelogs/10.7.0.md [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+- Fixed an issue where snapshots performed in XHR requests could reflect the
+  primary domain instead of current domain. Fixes
+  [#21496](https://github.com/cypress-io/cypress/issues/21496)
+- Introduced a new `GITHUB_RUN_ATTEMPT` environment variable when claiming specs
+  from Cypress Cloud to address a potential scenario that would cause invalid
+  Github Action requests to be accepted. Addressed in
+  [#23445](https://github.com/cypress-io/cypress/pull/23445)
+- Fixed an issue where an internal TypeScript type was exposed globally. Fixes
+  [#23333](https://github.com/cypress-io/cypress/pull/23333)
+- Fixed an issue where an incomplete `devServer` configuration would cause the
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:67]: Can we link to config option?
```suggestion
- Fixed an issue where an incomplete `component.devServer` configuration would cause the
```

File: content/_changelogs/10.7.0.md [hunk 21]
```diff
@@ -0,0 +1,81 @@
...
+  specs list to not display in the Cypress App. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/10.7.0.md:68]:
```suggestion
  specs list to not display in the Cypress App when running Component Tests in open mode. Fixes
```

File: content/_changelogs/10.7.0.md [hunk 22]
```diff
@@ -0,0 +1,81 @@
...
+  them in the future. Addresses
+  [#22933](https://github.com/cypress-io/cypress/issues/22933),
+  [#22934](https://github.com/cypress-io/cypress/issues/22934),
+  [#22798](https://github.com/cypress-io/cypress/issues/22798), and
+  [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the
+  "Create Spec from Component" capability introduced in v10.5.0. Addresses
+  [#23071](https://github.com/cypress-io/cypress/issues/23071)
+- Angular component testing projects can provide a custom configuration to the
+  Webpack dev server using the `projectConfig` key. This enables support for
```
[the remark this answers is not in the document]
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:33]: As in a link to the Webpack dev server configuration schema? Maybe this would be better phrased as "can provide configuration customizations to the Webpack dev server"

File: content/_changelogs/10.7.0.md (hunk 18 above)
Comment (emilyrohrbough) [reply content/_changelogs/10.7.0.md:61]: Seems like this is a new feature not a bug fix

Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:63]: You're probably right. My read on it is that it's presented as fixing an issue that users are encountering, but it's a new feature to help avoid the issue not really a fix itself

Comment (emilyrohrbough) [reply content/_changelogs/10.7.0.md:63]: no sure why rachel's feedback wasn't visible when I reviewed but actually think this is a bug fix after looking at code: does this make sense to you?
https://github.com/cypress-io/cypress-documentation/pull/4697#discussion_r958484638

Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:63]: Huh, okay. I'm glad you were able to make sense of that because I was really struggling. Thanks for the revised wording, will go with that

File: content/_changelogs/10.7.0.md (hunk 20 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md]: Added link

File: content/_changelogs/10.7.0.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md]: Added link to new doc section

File: content/_changelogs/10.7.0.md (hunk 19 above)
Comment (ZachJW34) [reply content/_changelogs/10.7.0.md]: No it was to use some new TS features internally, not related to node 18

File: content/_changelogs/10.7.0.md [hunk 23]
```diff
@@ -0,0 +1,86 @@
...
+
+- Display a default icon in the browser selection menu for browsers without
+  bundled icons. Fixes
+  [#21766](https://github.com/cypress-io/cypress/issues/21766).
+
+**Dependency Updates:**
+
+- Upgraded the bundled TypeScript version shipped with Cypress from `4.2.3` to
+  `4.7.4`. Addressed in
+  [#23523](https://github.com/cypress-io/cypress/pull/23523).
```
Comment (tbiethman) [inline content/_changelogs/10.7.0.md:86]: @rachelruderman mentioned this [in a comment](https://github.com/cypress-io/cypress-documentation/pull/4697#discussion_r957874422) but it might have gotten lost amongst the changes. This seems like it was an internal dependency bump and not user-facing, I don't think it needs to be in the changelog.

File: content/_changelogs/10.7.0.md (hunk 19 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md]: Thanks Zach. This sounds internal-only so I'll remove it

File: content/_changelogs/10.7.0.md (hunk 7 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md]: Seems like this is internal-only, so I'll go ahead and remove it

**Comment by mike-plummer** [comment]
Oops, merged Lachlan's PR into the changlog branch instead of the release branch 🤦

File: content/_changelogs/10.7.0.md (hunk 23 above)
Comment (emilyrohrbough) [reply content/_changelogs/10.7.0.md]: This did bump in @packages/server & webpack-dev-server so I do think this is user-facing -- it'll impact projusts using typescript right?

Comment (tbiethman) [reply content/_changelogs/10.7.0.md]: It could impact the types we distribute, but it's also 1. a change in minor version and 2. currently unpinned in those projects and will change again without our knowing.

I guess my thought is that if this is passive to users, it's not worth calling out. And if it's not passive, we would need to look into [versioning our types](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#version-selection-with-typesversions).

Comment (emilyrohrbough) [reply content/_changelogs/10.7.0.md]: oh I was looking at the MVB of ts-loader: https://github.com/cypress-io/cypress/pull/23523/files#diff-92fadee1dfb23b20d6b0f6557a3ea0b8a231a7591db73f9f37772383bd0575d9R184

File: content/_changelogs/10.7.0.md (hunk 8 above)
Comment (mike-plummer) [reply content/_changelogs/10.7.0.md:7]: Added blog post link from Jess

File: content/_changelogs/10.7.0.md (hunk 23 above)
Comment (ZachJW34) [reply content/_changelogs/10.7.0.md]: It could def affect the types we ship if we utilize types that are only available in newer versions of TS (like the `Awaited` utility type). Until we utilize these types, we should be ok with the bump.

@tbiethman make sense to create an issue in regards to better handling of our TS types?

**Review by BlueWinds — APPROVED** [review]

**Review by emilyrohrbough — APPROVED** [review]

File: content/_changelogs/10.7.0.md (hunk 23 above)
Comment (tbiethman) [reply content/_changelogs/10.7.0.md]: For reference, we currently document our minimum supported TS version as 3.4: https://docs.cypress.io/guides/tooling/typescript-support#Install-TypeScript

To cover our bases, it appears we'd need to:
1. version our types (where necessary) so that we continue to support our documented minimum spec
2. reset expectations for the minimum spec with each major Cypress release to drop support as needed

I'll get an issue logged to explore this more.
